CMakeLists.txt

```diff
@@ -132,6 +132,13 @@ if(CMAKE_SYSTEM_NAME STREQUAL Linux)
         clr_unknown_arch()
     endif()
     set(CLR_CMAKE_PLATFORM_LINUX 1)
+
+    # Detect Alpine Linux
+    file(READ "/etc/os-release" OS_RELEASE)
```

Comment:
Does this gracefully handle the case where this file doesn't exist (I don't know if there are any distros where that's the case)?